test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Verb.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+﻿Describe "Get-Verb" -Tags "CI" {
+
+    It "Should get a list of Verbs" {
+        Get-Verb | Should not be $null
+    }
+
+    It "Should get a specific verb" {
+        (Get-Verb -Verb Add | Measure-Object).Count | Should be 1
+        (Get-Verb -Verb Add -Group Common | Measure-Object).Count | Should be 1
+    }
+
+    It "Should get a specific group" {
+        Get-Verb -Group Common | Should not be $null
+    }
+
+    It "Should not return duplicate Verbs with Verb paramater" {
+        $dups = Get-Verb -Verb Add,ad*,a*
+        $unique = $dups | 
+            Select-Object -Property * -Unique
+        $dups.Count | Should be $unique.Count 
+    }
+
+    It "Should not return duplicate Verbs with Group paramater" {
+        $dupVerbs = Get-Verb -Group Data,Data
+        $uniqueVerbs = $dupVerbs | 
+            Select-Object -Property * -Unique
+        $dupVerbs.Count | Should be $uniqueVerbs.Count 
+    }
+
+    It "Should filter using the Verb parameter" {
+        Get-Verb -Verb fakeVerbNeverExists | Should BeNullOrEmpty
+    }
+
+    It "Should not accept Groups that are not in the validate set" {
+        try{
+            Get-Verb -Group FakeGroupNeverExists -ErrorAction Stop
```

Comment:
>Get-Verb -Group FakeGroupNeverExists -ErrorAction Stop [](start = 12, length = 54)

in case this doesn't throw, you need to add:
`throw "Expected error did not occur"`
or similar